src/components/common/Dialogs/NetworkContractUpgradeDialog/NetworkContractUpgradeDialogForm.tsx

```diff
@@ -86,7 +89,11 @@ const NetworkContractUpgradeDialogForm = ({
         </DialogSection>
       )}
       <DialogSection>
-        <ContractVersionSection currentVersion={version} />
+        <ContractVersionSection
+          colony={colony}
+          currentVersion={version}
+          colonyContractVersion={colonyContractVersion}
```

Comment:
Same question as above? 

Is it just because of the loading logic ?

Reply:
Yep, it's because of the loading logic for the spinner and submit button of this dialog. Otherwise, the submit button would be displayed as available even though the colony version hasn't loaded in.